timemachine/ff/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 import importlib_resources as resources
 
-from timemachine.constants import DEFAULT_PROTEIN_FF, DEFAULT_WATER_FF
+from timemachine.constants import DEFAULT_FF, DEFAULT_PROTEIN_FF, DEFAULT_WATER_FF
```

Comment:
super-nit, arguable: would it make sense for `DEFAULT_FF` to be defined here?

Reply:
Possibly.

I think it also makes sense as-is, and it is nice to have the top-level collection of defaults in `constants.py`... It doesn't seem too-cluttered as-is.

Reply:
Reviewed the contents of `constants.py`, and I think it still seems reasonable to me to leave DEFAULT_FF there https://github.com/proteneer/timemachine/pull/989/commits/4213c4cb1a47e7ad7cabb0d1e7f16869806d1919

However, what does look out-of-place in constants.py is these tags https://github.com/proteneer/timemachine/pull/989/commits/4213c4cb1a47e7ad7cabb0d1e7f16869806d1919#diff-58fa561ba38bbef66f3498a7dc546f317f49a1fb35ec5d5b667aebe9b8d74891R24-R26 , which look like an internal detail that doesn't need to be referenced much globally...

Reply:
These constants are only referenced in two files, `ff/handlers/serialize.py` and `ff/handlers/deserialize.py`.

(Moved these from top-level `constants.py` to a new file `ff/handlers/serialization_format.py` in https://github.com/proteneer/timemachine/pull/989/commits/075438b358637eaa88b43182b2b0c5cb789d58cb)